src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.ts

```diff
@@ -310,54 +311,10 @@ export class ProgrammingExerciseDetailComponent implements OnInit, OnDestroy {
         };
     }
 
-    getExerciseDetailsModeSection(exercise: ProgrammingExercise): DetailOverviewSection {
+    getExerciseDetailsBuildDetailsSection(exercise: ProgrammingExercise): DetailOverviewSection {
         return {
-            headline: 'artemisApp.programmingExercise.wizardMode.detailedSteps.difficultyStepTitle',
+            headline: 'artemisApp.programmingExercise.wizardMode.detailedSteps.buildDetailsStepTitle',
```

Comment:
Moved some of the details from the language section to the new `Build Details` section - the git diff makes that change look more complicated than it is 😅